npm/npm-debug-cli/cmd/parseIPtable.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright © 2021 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package cmd
+
+import (
+	"github.com/Azure/azure-container-networking/npm/npm-debug-cli/dataplaneParser/parser"
+	"github.com/spf13/cobra"
+)
+
+// parseIPtableCmd represents the parseIPtable command
+var parseIPtableCmd = &cobra.Command{
```

Comment:
your casing is different from your `convertIptableCmd`
I think good go style dictates `parseIPTableCmd` and `convertIPTableCmd`